mayastor/src/grpc/mayastor_grpc.rs

```diff
@@ -242,6 +246,110 @@ impl From<MayastorFeatures> for rpc::mayastor::MayastorFeatures {
     }
 }
 
+impl From<blk_device::BlockDevice> for BlockDevice {
+    fn from(b: blk_device::BlockDevice) -> Self {
+        Self {
+            devname: b.devname,
+            devtype: b.devtype,
+            devmajor: b.devmaj,
+            devminor: b.devmin,
+            model: b.model,
+            devpath: b.devpath,
+            devlinks: b.devlinks,
+            size: b.size,
+            partition: b.partition.map(block_device::Partition::from),
+            filesystem: b.filesystem.map(block_device::Filesystem::from),
+            available: b.available,
+        }
+    }
+}
+
+impl From<blk_device::FileSystem> for block_device::Filesystem {
+    fn from(fs: blk_device::FileSystem) -> Self {
+        let mountpoint = if fs.mountpoints.is_empty() {
+            String::from("")
+        } else {
+            fs.mountpoints.get(0).unwrap().to_string()
+        };
```

Comment:
might be able to tidy it up a bit but having something like this instead:
```suggestion
        let mountpoint = fs.mountpoints.get(0).unwrap_or_default();
```